### ISSUE TITLE
Documents blank if edited offline

### ISSUE THREAD

**Issue by jharej** [ONLYOFFICE/onlyoffice-owncloud#283]
Hi,

I've searched if anybody has had a similar issue but had no "luck". I've successfully installed Nextcloud and OnlyOffice (with docker). My problem is - online editing works, but if I open the same file directly from disk or download it from Nextcloud and open it in desktop version of Onlyoffice or Libreoffice, it is blank. If I export it directly from online onlyoffice, I can see the whole document. I've also first created a file offline, uploaded it to Nextcloud and edited it with online OnlyOffice. If I open it again offline, I can not see the changes made online. I've attached a screenshoot to present the issue. Am I missing something?

Thanks and bye
Janko 
![Izbor_103](https://user-images.githubusercontent.com/10975112/64648719-cd97e480-d41b-11e9-8b70-69f0f3a50188.png)

**Comment by alexanderonlyoffice** [comment]
Hello @jharej, please specify some additional information: 

1. OS of the server where ONLYOFFICE Document Server is deployed;
2. Its installation type (docker or deb/rpm);
3. Currently installed version of Document Server;
4. Version of Nextcloud;

It seems that you are downloading the file right after the file is closed. Please wait for a while and check if the new version of the file is saved in Nextcloud (you will see that the size of the file and the time when it was edited will change). 

Do you have any errors when clicking on the 'Save' button on integration app address settings page?

**Comment by jharej** [comment]
I've deployed the server on centos 7.6. Command used

 docker run -i -t -d -p 443:443 --restart=always --name=nameonlyoffice -v
/etc/letsencrypt/:/certificates  -v
/var/log/onlyoffice:/app/onlyoffice/DocumentServer/logs      -v
/var/www/onlyoffice/Data:/app/onlyoffice/DocumentServer/data      -v
/var/lib/onlyoffice:/app/onlyoffice/DocumentServer/lib     -v
/var/lib/postgresql:/app/onlyoffice/DocumentServer/db
onlyoffice/documentserver

Document editor version is 5.3.4
Nextcloud is version Nextcloud 16.0.3

thank and bye
Janko

V V sre., 11. sep. 2019 ob 11:51 je oseba Alexander Zozulya <
notifications@github.com> napisala:

**Comment by alexanderonlyoffice** [comment]
@jharej, please specify if you checked the scenario I indicated in the previous message: 

> It seems that you are downloading the file right after the file is closed. Please wait for a while and check if the new version of the file is saved in Nextcloud (you will see that the size of the file and the time when it was edited will change).

Do you get any errors?

> Do you have any errors when clicking on the 'Save' button on integration app address settings page?

**Comment by jharej** [comment]
No. No errors. I haven't downloaded the files prior to closing them. 

I've "solved" the problem by installing the newer version of app on nextcloud side and newest version of docker image. All the content that was edited in onlyoffice (just a few files thought) after installation vanished. So it is obvious that content edited in onlyoffice has not been send to nextcloud.

**Comment by alexanderonlyoffice** [comment]
@jharej,  

> No. No errors.

You mean on the new installation or you mean that there were no errors on the previous installation when clicking on the 'Save' button.

If you mounted all folders indicated in the installation guide during the previous installation, try to check the folder /app/onlyoffice/DocumentServer/lib/documentserver/App_Data/cache/ for the disappeared files.
